wagtail/admin/templates/wagtailadmin/generic/confirm_delete.html

```diff
@@ -6,5 +6,6 @@
     <form action="{{ view.get_delete_url }}" method="POST">
         {% csrf_token %}
         <input type="submit" value="{% trans 'Yes, delete' %}" class="button serious" />
+        <a href="{{ view.get_success_url }}" class="button button-secondary">{% trans "No, don't delete" %}</a>
```

Comment:
I'm surprised this wasn't here before.

I added this so that the snippet delete template can reuse as much of the template as possible, so that I can fully reuse this in #10072